wpilibj/src/athena/java/edu/wpi/first/wpilibj/Compressor.java

```diff
@@ -12,62 +12,67 @@
 import edu.wpi.first.wpilibj.tables.ITable;
 
 /**
- * Class for operating the PCM (Pneumatics compressor module) The PCM automatically will run in
- * close-loop mode by default whenever a Solenoid object is created. For most cases the Compressor
- * object does not need to be instantiated or used in a robot program. This class is only required
- * in cases where more detailed status or to enable/disable closed loop control. Note: you cannot
- * operate the compressor directly from this class as doing so would circumvent the safety provided
- * in using the pressure switch and closed loop control. You can only turn off closed loop control,
- * thereby stopping the compressor from operating.
+ * Class for operating a compressor connected to a PCM (Pneumatic Control Module). The PCM will
+ * automatically run in closed loop mode by default whenever a {@link Solenoid} object is created.
+ * For most cases, a Compressor object does not need to be instantiated or used in a robot program.
+ * This class is only required in cases where the robot program needs a more detailed status of the
+ * compressor or to enable/disable closed loop control.
+ *
+ * Note: you cannot operate the compressor directly from this class as doing so would circumvent
+ * the safety provided by using the pressure switch and closed loop control. You can only turn off
+ * closed loop control, thereby stopping the compressor from operating.
  */
 public class Compressor extends SensorBase implements LiveWindowSendable {
   private int m_compressorHandle;
   private byte m_module;
 
   /**
-   * Constructor.
+   * Makes a new instance of the compressor using the provided CAN device ID.  Use this constructor
+   * when you have more than one PCM.
    *
-   * <p>Most robots that use PCM will have a single module. Use this for supporting a second module
-   * other than the default.
-   *
-   * @param module The PCM CAN device ID.
+   * @param module The PCM CAN device ID (0 - 62 inclusive)
    */
   public Compressor(int module) {
     m_table = null;
```

Comment:
Yes, but we do this other places too.  Removing it is best left to a separate PR.